stdlib/LLVMLibUnwind_jll/StdlibArtifacts.toml

```diff
@@ -0,0 +1,143 @@
+[[LLVMLibUnwind]]
+arch = "aarch64"
+git-tree-sha1 = "d48f8172239e74a0cfd514289a90e1bd3f3b2e26"
+os = "macos"
+
+    [[LLVMLibUnwind.download]]
+    sha256 = "7668c3382f0d75b232e190e212f64f553ed102e15e8ebe4c3492239a401b7d52"
+    url = "https://github.com/JuliaBinaryWrappers/LLVMLibUnwind_jll.jl/releases/download/LLVMLibUnwind-v11.0.0+0/LLVMLibUnwind.v11.0.0.aarch64-apple-darwin.tar.gz"
```

Comment:
Looks like this needs to be updated

Reply:
After reviewing some other stdlib packages I think file shouldn't be committed at all